src/MetricExporter.ts

```diff
@@ -127,7 +127,7 @@ export class DynatraceMetricExporter implements MetricExporter {
 		}
 
 		const dtMetrics = metrics.map((metric) => {
-			const dimensions = Object.entries(metric.labels).map(([key, value]) => ({ key, value }));
+			const dimensions = Object.entries(metric.attributes).map(([key, value]) => ({ key, value }));
 			switch (metric.aggregator.kind) {
 				case AggregatorKind.SUM: {
 					// TODO: when metrics 0.20.0 is released use aggregator.aggregationTemporality
```

Comment:
Yes. Previously there was no aggregation temporality. Now that there is, we will need to handle both cases (delta and cumulative). Because the metric sdk is in the process of being rewritten I don't want to spend too much effort on this version because it will be changed significantly so for now we're just going to drop the cumulative counter as it is not supported by dynatrace. After the new SDK is released I will update the exporter to support those counters.